contrib/systemd-with-workers/system/matrix-synapse-worker@.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Synapse Matrix Worker
+After=matrix-synapse.service
+BindsTo=matrix-synapse.service
+
+[Service]
+Type=forking
```

Comment:
shouldn't this be `simple` ?

Reply:
Yes, must have slipped through when I was copying stuff. Fixing now...

Reply:
Fixed